src/sourcemaps/tools/vite.ts

```diff
@@ -0,0 +1,96 @@
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack, { select } from '@clack/prompts';
+import chalk from 'chalk';
+import {
+  abortIfCancelled,
+  getPackageDotJson,
+  hasPackageInstalled,
+  installPackage,
+} from '../../utils/clack-utils';
+
+import {
+  SourceMapUploadToolConfigurationFunction,
+  SourceMapUploadToolConfigurationOptions,
+} from './types';
+
+const getCodeSnippet = (options: SourceMapUploadToolConfigurationOptions) =>
+  chalk.gray(`
+import { defineConfig } from "vite";
+${chalk.greenBright('import { sentryVitePlugin } from "@sentry/vite-plugin"')};
+
+export default defineConfig({
+  build: {
+    ${chalk.greenBright(
+      'sourcemap: true, // Source map generation must be turned on',
+    )}
+  },
+  plugins: [
+    // Put the Sentry vite plugin after all other plugins
+    ${chalk.greenBright(`sentryVitePlugin({
+      authToken: process.env.SENTRY_AUTH_TOKEN,
+      org: "${options.orgSlug}",
+      project: "${options.projectSlug}",${
+      options.selfHosted ? `\n      url: "${options.url}",` : ''
+    }
+    }),`)}
+  ],
+});
+`);
+
+export const configureVitePlugin: SourceMapUploadToolConfigurationFunction =
+  async (options) => {
+    await installPackage({
+      packageName: '@sentry/vite-plugin',
+      alreadyInstalled: hasPackageInstalled(
+        '@sentry/vite-plugin',
+        await getPackageDotJson(),
+      ),
+    });
+
+    clack.log.step(
+      `Add the following code to your ${chalk.bold('vite.config.js')} file:`,
+    );
+
+    // Intentially logging directly to console here so that the code can be copied/pasted directly
+    // eslint-disable-next-line no-console
+    console.log(getCodeSnippet(options));
+
+    const copiedConfigSnippet = await select({
+      message: 'Did you copy the snippet above?',
+      options: [{ label: 'Yes, continue!', value: true }],
+      initialValue: true,
+    });
+    abortIfCancelled(copiedConfigSnippet);
+
+    clack.log.step(
+      'Add the Sentry auth token as an environment variable or secret to your CI setup:',
+    );
+
+    // Intentially logging directly to console here so that the code can be copied/pasted directly
+    // eslint-disable-next-line no-console
+    console.log(
+      chalk.greenBright(`
+SENTRY_AUTH_TOKEN=${options.authToken}
+`),
+    );
+
+    clack.log.warn(
+      chalk.redBright('DO NOT commit this auth token to your repository!'),
```

Comment:
l: The bright red with green from the auth token and the yellow from the warning is a big jarring to the eye. Do we want to change this from red to yellow?

Reply:
Better :D
![image](https://github.com/getsentry/sentry-wizard/assets/8420481/c6d2d688-15b8-4173-9a27-d04bba1bdcbf)